CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ This is the log of notable changes to the Expo client that are developer-facing.
 
 ### 📚 3rd party library updates
 
+### 🛠 Breaking changes
+
+### 🎉 New features
+
+### 🐛 Bug fixes
+
+- Fixed multiplied callbacks in av after replaying ([#7193](https://github.com/expo/expo/pull/7193) by [@mczernek](https://github.com/mczernek))
+
+## 37.0.0
```

Comment:
IIRC adding new SDK section is done in release cycle.
Please remove it from here 😊